NEWS.md

```diff
@@ -1,6 +1,13 @@
 # rhino (development version)
 
 1. New `box_alphabetical_imports_linter` checks if all imports are alphabetically sorted.
+1. Update GitHub Workflow template triggers. To update your workflow run:
+    ```r
+    file.copy(
+      system.file("templates", "github_ci", "dot.github", "workflows", "rhino-test.yml", package = "rhino"),
+      file.path(".github", "workflows", "rhino-test.yml")
+    )
+    ```
```

Comment:
A little bit of nitpicking:
- the rest of the NEWS.md file uses actual numbers instead of relying on markdown. I'm not saying that this is the best choice (pros - you still got the numbers in the "raw" version), but please follow the existing convention.
- As the migration guides for the previous releases are added as vignettes, it makes sense to follow this pattern. My suggestion here is to update the CONTRIBUTION.md release section - the person responsible for the release should add a migration guide before pushing a new version of Rhino.

Reply:
Numbering changed 👌

I was not sure where to put the code so it won't get lost. I agree that the person responsible for the release should put together a migration guide. However, I think it's contributors who should be aware whether their changes need an entry in the migration guide.

I think NEWS.md is fine to have hints of how to migrate **after a particular change**. Especially that NEWS.md is _the_ source of truth for people using the development version. These later can be gathered into a migration guide vignette.

With that in mind I would update CONTRIBUTING.md with action points for both parties:
- Contributors: update NEWS and describe how to migrate after the change
- Releaser: put together a migration guide

Thoughts?

Reply:
I'm ok with this approach. I would add a mention that the releaser might decide to remove some/all migration descriptions from the NEWS.md.